.clj-kondo/marick/midje/config.edn

```diff
@@ -0,0 +1,80 @@
+{:hooks {:analyze-call {midje.sweet/tabular marick.midje/tabular}}
```

Comment:
@borkdude do you know if committing these configs here in combination with tagging `midje` as `:scope "provided"` in the `matcher-combinators` deps mean that the `midje` configs for `clj-kondo` would be still be picked up via `clj-kondo --lint $(clojure -Spath) --dependencies --copy-configs` ?

Like does `--copy-configs` copy all `.clj-kondo` configs from a lib even if it is from a different org namespace?